CHANGELOG.md

```diff
@@ -13,11 +27,17 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- release 6.2.0 [\#388](https://github.com/voxpupuli/puppet-r10k/pull/388) ([bastelfreak](https://github.com/bastelfreak))
 - Add logging messages [\#385](https://github.com/voxpupuli/puppet-r10k/pull/385) ([amateo](https://github.com/amateo))
 
 ## [v6.1.0](https://github.com/voxpupuli/puppet-r10k/tree/v6.1.0) (2017-06-24)
+
 [Full Changelog](https://github.com/voxpupuli/puppet-r10k/compare/v6.0.0...v6.1.0)
 
+**Breaking changes:**
```

Comment:
???

Reply:
latest changelog generator now supports a section for stuff tagges with backwards-incompatible

Reply:
But releasing 6.0.0 is not a breaking change of 6.1.0

Reply:
you added that label to the corresponding PR :P I removed it and regenerated the changelog

Reply:
Do I didn’t! Why am I getting the blame for everything today! :)